Dockerfile

```diff
@@ -25,6 +25,7 @@ RUN mkdir out \
 
 # Use a runtime image based on Debian slim
 FROM debian:10.6-slim
+RUN apt-get update && apt-get install -y ca-certificates && rm -rf /var/lib/apt/lists/*
```

Comment:
We discussed this, but tl;dr for others -- it's so that the OIDC discovery can have access to a default CA bundle, because Debian `-slim` images don't have one.